templates/todo/projects/nodejs-mongo-aks/.repo/bicep/infra/hooks/predeploy.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+echo "Retrieving cluster credentials"
+az aks get-credentials --resource-group ${AZURE_RESOURCE_GROUP} --name ${AZURE_AKS_CLUSTER_NAME}
```

Comment:
Templates should not take a dependency on az cli.

Reply:
Removed